src/compas_cgal/meshing.py

```diff
@@ -1,15 +1,21 @@
+from typing import List, Tuple
 import numpy as np
 from compas_cgal._cgal import meshing
 from compas.plugins import plugin
 
+from compas_cgal.types import VerticesFaces, VerticesFacesNumpy
+
 
 __all__ = [
     'remesh'
 ]
 
 
 @plugin(category='trimesh', pluggable_name='trimesh_remesh')
-def remesh(mesh, target_edge_length, number_of_iterations=10, do_project=True):
+def remesh(mesh: VerticesFaces,
+           target_edge_length: float,
+           number_of_iterations: int = 10,
+           do_project: bool = True) -> VerticesFacesNumpy:
```

Comment:
isn't it `bool=True` and `int=10` without spaces around `=`?

Reply:
apparently not in the case of using type annotations